parity-crypto/src/publickey/ec_math_utils.rs

```diff
@@ -37,52 +37,46 @@ lazy_static! {
 	pub static ref CURVE_ORDER: U256 = H256::from_slice(&SECP256K1_CURVE_ORDER).into_uint();
 }
 
-/// Whether the public key is valid.
-pub fn public_is_valid(public: &Public) -> bool {
```

Comment:
do you remember what was that about, also worth mentioning in the changelog breaking changes

Reply:
I'll dig up the details but essentially it is now impossible to construct an invalid public key so this is not needed.

Reply:
So this happened [here](https://github.com/rust-bitcoin/rust-secp256k1/pull/37) and the CHANGELOG says: "Remove PublicKey::new() and PublicKey::is_valid() since new was unsafe and it should now be impossible to create invalid PublicKey objects through the API"

Reply:
ok, thanks for digging, let's add this to the changelog, we were exporting this function previously